source/tutorials/first-steps/nix-language.md

```diff
@@ -1861,12 +1861,12 @@ It is an error if the network request fails.
 
 A build task in Nix is called a *derivation*.
```

Comment:
This description should probably be changed as I suspect many folks, myself included, don't know what a "build task" is and so the definition feels circularly incomplete. As if a dictionary said:

Derivation (n) : See _build task_

But then there was no entry for _build task_

Reply:
```suggestion
```

Let's remove it then.